source/Fleet.cpp

```diff
@@ -90,8 +90,8 @@ void Fleet::Load(const DataNode &node)
 		else if(key == "fighters" && hasValue)
 		{
 			fighterNames = GameData::Phrases().Get(child.Token(1));
-			child.PrintTrace("Warning: Deprecated use of \"fighters\" <names>."
-				" Use \"names\" <names> node under \"fighters\" instead.");
+			if(child.HasChildren())
+				child.PrintTrace("Warning: Skipping unused children of \"fighters\" tag:");
```

Comment:
I agree it's more elegant this way but shouldnt we rather take the child? (bc there being a child seems more intended than just a typo there may be here?)
not really important though, just thought I'd mention it

Reply:
idk

```
fighters "names"
    personality
        timid
```

seemed weird to me? idk

Reply:
I mostly meant that
```
fighters s
    personality
        timid
```
is more likely to be a mistake (and therefore the s should be ignored)
whereas
```
fighters "fighters names"
       p
```
feels more unlikely as an error to me

but yeah I'd just parse the personality or new fighter name anyway if there is one, regrouping the two ifs into one if(fighters)

but then again it works like you did it too

Reply:
We should accept the name being in-line and the personality being a child. Now if for some reason the name is in both locations, the child name should take precedence. This is how ship bay hardpoints function; originally, the zoom of an engine's sprite was only defined in-line as `fighter <x> <y> <zoom>`, but even though we now have a `zoom` child of `bay`, the in-line syntax is still accepted, while the child syntax takes precedence if both exist.